src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -19,6 +18,8 @@ internal static partial class Interop
 {
     internal static partial class OpenSsl
     {
+        private const string DisableTlsResumeCtxSwitch = "System.Net.Security.DisableTlsResume";
```

Comment:
AppContext switch values usually start with "Switch."

```suggestion
        private const string DisableTlsResumeCtxSwitch = "Switch.System.Net.Security.DisableTlsResume";
```